benchmark/bench-report/bin/bench-runner.sh

```diff
@@ -39,6 +39,12 @@ print_help () {
   echo
   echo "--benchmarks: benchmarks to run, use 'help' for list of benchmarks"
   echo "--compare: compare the specified benchmarks with each other"
+
+  echo "--diff-cutoff-percent: Diff percentage used for benchmark selection."
+  echo "The diff is considered only between the first and second column of the"
```

Comment:
Not very clear description. How about something like;

"When comparing benchmarks with a baseline, show only those benchmarks that changed by more than the cutoff percentage. When more than two sets of benchmarks are present, this applies to only the first two."